src/Domain.LinnApps/Reports/PurchaseOrdersReportService.cs

```diff
@@ -386,7 +391,8 @@ private static void AddPartReportColumns(SimpleGridLayout reportLayout)
             PurchaseOrderDetail orderDetail,
             PurchaseOrderDelivery delivery,
             decimal ledgerQty,
-            string currencyCode)
+            string currencyCode,
+            bool isFirstDelivery)
```

Comment:
do you need this isFirstDelivery bool? Doesn't the delivery itself tell you whether its the first delivery or not?